addons/mail/models/mail_thread.py

```diff
@@ -2885,8 +2894,21 @@ def _message_create(self, values_list):
             for x in ('from', 'to', 'cc'):
                 create_values.pop(x, None)
             create_values['partner_ids'] = [Command.link(pid) for pid in (create_values.get('partner_ids') or [])]
+            channel_ids = create_values.pop('channel_ids', None)
             create_values_list.append(create_values)
 
+            # Create message in mentioned channels
+            if channel_ids:
+                user = f'<a href="/odoo/res.partner/{self.env.user.partner_id.id}">{self.env.user.name}</a>'
```

Comment:
We don't want to hard-code a link like that, it should use the oe-model, oe-id trick so that we can adapt it in future versions without having to update the whole message table.